src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -11380,11 +11695,27 @@ static bool TakeOutParam<T>(T y, out T x)
 }
 ";
             var compilation = CreateCompilationWithMscorlib45(source, options: TestOptions.DebugExe, parseOptions: TestOptions.Regular);
-            CompileAndVerify(compilation, expectedOutput: @"throw");
+
+            compilation.VerifyDiagnostics(
+                // (15,9): error CS0103: The name 'x1' does not exist in the current context
+                //         x1++;
+                Diagnostic(ErrorCode.ERR_NameNotInContext, "x1").WithArguments("x1").WithLocation(15, 9)
+                );
+
+
+            var tree = compilation.SyntaxTrees.Single();
+            var model = compilation.GetSemanticModel(tree);
+
+            var x1Decl = GetOutVarDeclarations(tree, "x1").ToArray();
+            var x1Ref = GetReferences(tree, "x1").ToArray();
+            Assert.Equal(1, x1Decl.Length);
+            Assert.Equal(1, x1Ref.Length);
+            VerifyModelForOutVar(model, x1Decl[0]);
+            VerifyNotInScope(model, x1Ref[0]);
         }
 
         [Fact]
-        public void Scope_Using_01()
+        public void Scope_ThrowStatement_03()
```

Comment:
Ah, I see the throw in the speculation.

---
In reply to: [75794461](https://github.com/dotnet/roslyn/pull/13273#discussion_r75794461) [](ancestors = 75794461)